js/addTask.js

```diff
@@ -0,0 +1,75 @@
+import { putRequest } from "./putRequest.js";
+import { deleteRequest } from "./deleteRequest.js";
+import { updateTaskCount } from "./updateTaskCount.js";
+import { updateLocalStorage } from "./updateLocalStorage.js";
+import { removeFromLocalStorage } from "./removeFromLocalStorage.js";
+
+export function addTask(todo, todoTableBody) {
+    const row = document.createElement('tr');
+    row.innerHTML = `
+        <td>${todo.id}</td>
+        <td class="todo-text ${todo.completed ? 'completed' : ''}" contenteditable="false">
+            ${todo.todo}
+        </td>
+        <td>${todo.userId}</td>
+        <td class="status ${todo.completed ? 'completed-status' : 'pending-status'}">
+            ${todo.completed ? 'Completed' : 'Pending'}
+        </td>
+        <td>
+            <button class="edit-btn" data-id="${todo.id}">Edit</button>
```

Comment:
Since you need to create multiple buttons, try to create a function to handle button creation:

```
function createButton(className, text, id, clickHandler) {
    const button = document.createElement("button");
    button.className = className;
    button.textContent = text;
    button.dataset.id = id;
    button.addEventListener("click", clickHandler);
    return button;
}
```

Current function does multiple things, that's why I want you to extract logic into separate functions.